tests/test_smart_completion_public_schema_only.py

```diff
@@ -302,7 +314,7 @@ def test_auto_escaped_col_names(completer, complete_event):
     result = list(completer.get_completions(Document(text=text, cursor_position=position), complete_event))
     assert result == [
         Completion(text="*", start_position=0),
-        Completion(text="`ABC`", start_position=0),
+        Completion(text="ABC", start_position=0),
```

Comment:
Previous tests were escaping the upper case column names and not sure why tjough. If there is a solid case, we can close this PR. I don't see any issue of passing upper case as is to sqlite3.

```sql
sqlite> .open :memory;
sqlite> create table TEST(ID int);
sqlite> insert into TEST(1);
Parse error: near "1": syntax error
  insert into TEST(1);
                   ^--- error here
sqlite> insert into TEST values(1);
sqlite> select ID from TEST;
1
sqlite>
```